src/pages/starter-kit/checkout/observability.md

```diff
@@ -0,0 +1,173 @@
+
+   ```javascript
+   import {
+     defineTelemetryConfig,
+     getAioRuntimeResource,
+     getPresetInstrumentations,
+     // localCollectorConfig, // optional helper if you want a local collector
+     // other helpers from '@adobe/aio-lib-telemetry' as needed
+   } from '@adobe/aio-lib-telemetry';
+   
+   // Exporter wiring helper (replace with your service-specific config; refer to use cases documentation)
+   function buildExporters(params) {
+     return {
+       // Logs: add your log record processor + exporter instance here
+       // logRecordProcessors: params.OTLP_ENDPOINT ? [buildYourLogExporter(params)] : [],
+   
+       // Traces: uncomment and provide span processor/exporter if you enable tracing
+       // spanProcessors: params.OTLP_TRACES_ENDPOINT ? [buildYourTraceExporter(params)] : [],
+   
+       // Metrics: provide metrics exporters if metrics collection is enabled
+       // metricsExporters: [],
+     };
+   }
+   
+   const telemetryConfig = defineTelemetryConfig((params, isDev) => ({
+     sdkConfig: {
+       serviceName: 'commerce-checkout-starter-kit',
+       instrumentations: getPresetInstrumentations('simple'),
+       resource: getAioRuntimeResource(),
+   
+       // Optional: uncomment for local collector convenience
+       // ...localCollectorConfig(),
+   
+       // Insert log / trace / metric exporter objects via helper
+       ...buildExporters(params),
+     },
+     // Set true only when debugging telemetry setup
+     diagnostics: false,
+   }));
+   
+   export { telemetryConfig };
+   ```
+
+1. Surface endpoint and auth inputs in [`app.config.yaml`](https://github.com/adobe/commerce-checkout-starter-kit/blob/main/app.config.yaml), as shown in the following excerpt. (You can also use secrets.)
+
+   ```yaml
+   actions:
+     collect-taxes:
+     inputs:
+        ENABLE_TELEMETRY: true
+        OTLP_ENDPOINT: $OTLP_ENDPOINT        # e.g. https://otel.example.com/v1/logs
+        OTLP_API_KEY: $OTLP_API_KEY          # token/key if service requires
+        # repeat for other actions or centralize pattern
+   ```
+
+1. Update your local development `.env` file with the appropriate endpoint and auth values. Example for a local OpenTelemetry collector:
+
+   ```text
+   OTLP_ENDPOINT=http://localhost:4318
+   OTLP_API_KEY=LOCAL_DEV_KEY_OR_EMPTY
+   ```
+
+For concrete exporter code (such as constructing OTLP exporters, using batch processors, service specific headers), refer to:
+
+- [Use cases (configuration examples)](https://github.com/adobe/aio-lib-telemetry/tree/main/docs/use-cases)
+- [Exporting data guide](https://github.com/adobe/aio-lib-telemetry/blob/main/docs/usage.md#exporting-data)
+
+## Troubleshooting
+
+If you encounter connection errors when exporting telemetry data:
+
+```text
```

Comment:
Consider use ` instead of ```, because the error messages are cut. This block is not code demonstation.
<img width="705" height="178" alt="Screenshot 2025-11-19 at 09 51 00" src="https://github.com/user-attachments/assets/3f5f8ab6-25c4-42af-a67c-a94fc43b70b4" />